include/pcp/octree/linked_octree_node.hpp

```diff
@@ -39,9 +39,6 @@ struct octree_parameters_t
     aabb_type voxel_grid{};            ///< The octree's bounding box
 };
 
-template <class Element, class ParamsType>
```

Comment:
My only question/concern, why did you remove the template of the iterator in the node?

Reply:
Because the declaration + definition of the linked_octree_iterator is already done in the "linked_octree_iterator.hpp" file that is included at the top of the file. This declaration is redundant.